src/Skoruba.IdentityServer4.Admin/Skoruba.IdentityServer4.Admin.csproj

```diff
@@ -13,7 +15,8 @@
 		<PackageReference Include="IdentityServer4.EntityFramework" Version="2.3.2" />
 		<PackageReference Include="Microsoft.EntityFrameworkCore.Tools.DotNet" Version="2.0.3" />
 		<PackageReference Include="Microsoft.Extensions.Options" Version="2.2.0" />
-		<PackageReference Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="2.2.0" />
+    <PackageReference Include="Microsoft.VisualStudio.Azure.Containers.Tools.Targets" Version="1.0.2105168" />
```

Comment:
What is this package used for? @Seaear

Reply:
This package is required for VS 2017 to build Docker